lib/reporter/reporters/html.js

```diff
@@ -224,10 +228,8 @@ class HtmlReporter extends BaseReporter {
   async writeReport() {
     const results = this.adaptResults();
 
-    const output_folder = this.options.output_folder;
-    const shouldOpenReport = this.options.openReport;
-
-    const destFolder = path.join(output_folder, this.getFolderPrefix(), 'nightwatch-html-report');
+    const {output_folder, openReport: shouldOpenReport, folder_format} = this.options;
+    const destFolder = path.join(output_folder, this.getFolderPrefix(folder_format), 'nightwatch-html-report');
```

Comment:
why do you need `folder_format` here?

Reply:
Thanks for pointing it out; we don't need it. I missed this during refactor